habitat/plan.sh

```diff
@@ -0,0 +1,21 @@
+pkg_name=ci-studio-common
+pkg_origin=chef
+pkg_version="0.1.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="Shared helpers for use inside CIs (like Travis) and a Habitat Studio"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_deps=(
+  core/busybox
```

Comment:
Im not sure we wanna add busybox entirely. I think we can be a bit smart and just install what's needed. otherwise all our studios will grow with things we might not use.

Reply:
cc/ @schisamo @tduffield @jonsmorrow ^